ui/src/app/components/admin/audit-card-list-view.tsx

```diff
@@ -0,0 +1,131 @@
+import colors from 'app/styles/colors';
+import * as fp from 'lodash/fp';
+import * as moment from 'moment';
+import * as React from 'react';
+import {
+  AuditAction,
+  AuditEventBundle,
+  AuditEventBundleHeader,
+  AuditTargetPropertyChange
+} from '../../../generated';
+import {ActionAuditCardBase} from '../card';
+import {FlexRow} from '../flex';
+
+
+const HidableCell = (props: {content: string}) => {
+  const {content} = props;
+  return <div style={{backgroundColor: content ? 'white' : '#f0f3f5',
+    border: '1px solid',
+    boxSizing: 'border-box'}}>{content}</div>;
+};
+
+const PropertyChangeListEntry = (props: {targetProperty?: string, previousValue?: string, newValue?: string}) => {
+  const {targetProperty, previousValue, newValue} = props;
+  return <React.Fragment>
+    <HidableCell content={targetProperty}/>
+    <HidableCell content={previousValue}/>
+    <HidableCell content={newValue}/>
+  </React.Fragment>;
+};
+
+const PropertyChangeListView = (props: { propertyChanges: AuditTargetPropertyChange[] }) => {
+  const {propertyChanges} = props;
+  const propertyCellStyle = {fontWeight: 600, border: `1px solid`};
+
+  return propertyChanges.length > 0 ?
+  <div style={{
+    margin: '0.25rem 0 0rem 1rem',
```

Comment:
nit: either put `rem` everywhere for consistency or break it out into margin-top and margin-left or whatever this is